ERROR - 2024-03-07 19:03:47 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u954703204_donaji'@'localhost' (using password: YES) in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:03:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre_usuario` = :nombre_usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->loggeo()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:07:07 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u954703204_donaji'@'187.228.136.184' (using password: YES) in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('cinetickett.com', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'187.228.136.184' (using password: YES) in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:07:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'187.228.136.184' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre_usuario` = :nombre_usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->loggeo()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:07:57 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('http://cinetick...', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#7 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('http://cinetick...', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:07:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre_usuario` = :nombre_usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->loggeo()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:07:59 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('http://cinetick...', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#7 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('http://cinetick...', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:07:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre_usuario` = :nombre_usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->loggeo()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:08:20 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u954703204_donaji'@'187.228.136.184' (using password: YES) in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('191.101.79.91', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'187.228.136.184' (using password: YES) in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:08:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'187.228.136.184' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre_usuario` = :nombre_usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->loggeo()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:14:50 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for  191.101.79.52 failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect(' 191.101.79.52', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#7 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for  191.101.79.52 failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect(' 191.101.79.52', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for  191.101.79.52 failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\cine\app\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Login->loggeo()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:14:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for  191.101.79.52 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre_usuario` = :nombre_usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(19): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->loggeo()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:30:52 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:30:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
ORDER BY `id_usuario` ASC', [], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:30:58 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:30:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
ORDER BY `id_usuario` ASC', [], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:30:58 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:30:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
ORDER BY `id_usuario` ASC', [], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:30:58 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:30:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
ORDER BY `id_usuario` ASC', [], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:30:58 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:30:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
ORDER BY `id_usuario` ASC', [], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-03-07 19:39:22 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 185)
#1 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#8 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#9 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#10 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\cine\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('srv867.hstgr.io', 'u954703204_dona...', 'X=Uf8TJz9~', 'u954703204_cine', 3306, '', 0)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#8 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#9 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido.  in C:\laragon\www\cine\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\cine\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\cine\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\cine\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\cine\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\cine\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\cine\app\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\cine\system\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
#7 C:\laragon\www\cine\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\laragon\www\cine\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\cine\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-03-07 19:39:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for srv867.hstgr.io failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
ORDER BY `id_usuario` ASC', [], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Usuarios.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:49:53 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:51:08 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:51:11 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:51:12 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:51:15 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:51:17 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:51:18 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:51:22 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 55.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:58:11 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 54.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:59:03 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPreviousPage()
in APPPATH\Views\usuarios\crud_usuarios.php on line 54.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 21:59:10 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPreviousPage()
in APPPATH\Views\usuarios\crud_usuarios.php on line 54.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:00:50 --> ErrorException: foreach() argument must be of type array|object, string given
in APPPATH\Views\usuarios\crud_usuarios.php on line 67.
 1 APPPATH\Views\usuarios\crud_usuarios.php(67): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, string given', 'C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php', 67)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 5 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:00:51 --> ErrorException: foreach() argument must be of type array|object, string given
in APPPATH\Views\usuarios\crud_usuarios.php on line 67.
 1 APPPATH\Views\usuarios\crud_usuarios.php(67): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, string given', 'C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php', 67)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 5 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:00:52 --> ErrorException: foreach() argument must be of type array|object, string given
in APPPATH\Views\usuarios\crud_usuarios.php on line 67.
 1 APPPATH\Views\usuarios\crud_usuarios.php(67): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, string given', 'C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php', 67)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 5 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:00:53 --> ErrorException: foreach() argument must be of type array|object, string given
in APPPATH\Views\usuarios\crud_usuarios.php on line 67.
 1 APPPATH\Views\usuarios\crud_usuarios.php(67): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, string given', 'C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php', 67)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 5 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:02:22 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPreviousPage()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:20 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:23 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:24 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:25 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:26 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:27 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:29 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:30 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:31 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:03:32 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 57.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:05:01 --> Error: Call to undefined method CodeIgniter\Pager\Pager::hasPrevious()
in APPPATH\Views\usuarios\crud_usuarios.php on line 59.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 4 APPPATH\Controllers\Usuarios.php(38): view('usuarios/crud_usuarios', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:41:49 --> CodeIgniter\Pager\Exceptions\PagerException: "template_name" is not a valid Pager template.
in SYSTEMPATH\Pager\Pager.php on line 122.
 1 SYSTEMPATH\Pager\Pager.php(122): CodeIgniter\Pager\Exceptions\PagerException::forInvalidTemplate('template_name')
 2 SYSTEMPATH\Pager\Pager.php(112): CodeIgniter\Pager\Pager->displayLinks('default', 'template_name')
 3 APPPATH\Controllers\Usuarios.php(24): CodeIgniter\Pager\Pager->makeLinks(2, 10, 28, 'template_name')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:42:13 --> ErrorException: Undefined variable $pager_links
in APPPATH\Views\usuarios\crud_usuarios.php on line 54.
 1 APPPATH\Views\usuarios\crud_usuarios.php(54): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pager_links', 'C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php', 54)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 5 APPPATH\Controllers\Usuarios.php(37): view('usuarios/crud_usuarios', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:42:54 --> ErrorException: Undefined variable $pager_links
in APPPATH\Views\usuarios\crud_usuarios.php on line 54.
 1 APPPATH\Views\usuarios\crud_usuarios.php(54): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pager_links', 'C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php', 54)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\cine\\app\\Views\\usuarios\\crud_usuarios.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('usuarios/crud_usuarios', [], true)
 5 APPPATH\Controllers\Usuarios.php(37): view('usuarios/crud_usuarios', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-03-07 22:45:57 --> ErrorException: Attempt to read property "pager" on array
in APPPATH\Controllers\Usuarios.php on line 37.
 1 APPPATH\Controllers\Usuarios.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "pager" on array', 'C:\\laragon\\www\\cine\\app\\Controllers\\Usuarios.php', 37)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->usuarios_view()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
